src/hackyballs/controlpanel.js

```diff
@@ -33,17 +32,8 @@ var HBControlPanel = GObject.registerClass({
         this._model = model;
     }
 
-    _onUnlockStateChanged(win) {
-        const [, l2] = win.getUnlockState();
-        if (l2)
-            this._level2lock.locked = false;
-    }
-
     bindWindow(win) {
-        win.connect('unlock-state-changed', this._onUnlockStateChanged.bind(this));
-        this._onUnlockStateChanged(win);
-
-        // FIXME For playtest only. Remove this later.
-        this._level2lock.connect('overlay-clicked', () => win.unlock());
+        win.getMainLockscreen().key = 'item.key.fizzics.1';
+        this._level2lock.key = 'item.key.fizzics.2';
```

Comment:
Could probably be set at construction, no? (also for other lockscreens)

Reply:
A couple reasons for doing it like this: a) we can't do it for the window's lockscreen, b) like it happens with the framework lockscreens, some times we need to define some rules before the locks can actually be unlocked (so its simple assign keys only after we have all the lockscreens available), and c) I think is just easier to read to have all those strings in one place.